.travis.yml

```diff
@@ -3,19 +3,25 @@ language: php
 php: [5.4, 5.5, 5.6, hhvm]
 
 matrix:
-  allow_failures:
-    - php: hhvm
-
-cache:
```

Comment:
cache feature is only available for paid plan (http://docs.travis-ci.com/user/caching/)